application-analytics-ui/src/main/resources/Analytics/Code/Macros/PagesFollowingASiteSearch.xml

```diff
@@ -268,7 +269,7 @@
       }
     }
   })
-  #getAnalyticsMacroId('PagesFollowingASiteSearch', $macroId)
+  #getAnalyticsMacroId('pagesFollowingASiteSearchMacro', $macroId)
```

Comment:
Why you added the macro in the id? It does not seem necessary

Reply:
I added it for consistency with the other analytics macros, and for clarity. However, if you think it's redundant, I can remove it.

Reply:
Yes, for me it seems redundant